rs-script/Cargo.toml

```diff
@@ -17,3 +17,12 @@ num-traits = "0.2"
 ndarray = "0.15.6"
 itertools = "0.13.0"
 rayon = "1.10.0"
+tracing = "0.1.40"
+tracing-subscriber = { version = "0.3.17", features = ["std", "env-filter"] }
+tracing-forest = { version = "0.1.6", features = ["ansi", "smallvec"] }
+concrete-ntt = { git = "https://github.com/zama-ai/concrete-ntt", features = [
```

Comment:
@fhedude, fhe.rs already implements NTT, no? Perhaps we can use fhe.rs' implementation to avoid adding another dependency?